continunet/tests/test_unet.py

```diff
@@ -0,0 +1,91 @@
+"""Tests for the UNet model."""
+
+import pytest
+
+from continunet.network.unet import Unet
+
+
+class TestUnet:
+    """Tests for the UNet model."""
+
+    model = Unet
+
+    def test_build_model(self, input_shape):
+        """Test the compile_model method"""
+
+        test_model = self.model(input_shape).build_model()
+
+        assert test_model is not None
+        assert test_model.input_shape == (None, 256, 256, 1)
+        assert test_model.output_shape == (None, 256, 256, 1)
+        assert len(test_model.layers) == 49
```

Comment:
These are good, but what if you change your fixture size? Then you need to change the test in two places. Might be better to check against the fixture parameter to the test, so that there's a single source of truth as it were.

Reply:
I've come up with a pretty janky solution to this because I can't remember how to do this nicely with numpy